libs/foundations/tokens.json

```diff
@@ -6930,25 +6706,25 @@
         },
         "hero": {
```

Comment:
health care related images are made in the component layer - therefore the images are not showing up on the Saddles theme.